systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfST.java

```diff
@@ -112,8 +112,9 @@ void testSimpleDynamicConfiguration() {
 
         updateAndVerifyDynConf(Environment.TEST_SUITE_NAMESPACE, testStorage.getClusterName(), deepCopyOfShardKafkaConfig);
 
-        kafkaConfigurationFromPod = KafkaCmdClient.describeKafkaBrokerUsingPodCli(Environment.TEST_SUITE_NAMESPACE, scraperPodName, KafkaResources.plainBootstrapAddress(testStorage.getClusterName()), podNum);
-        assertThat(kafkaConfigurationFromPod, containsString("unclean.leader.election.enable=" + true));
+        // Wait until the configuration is properly set and returned by Kafka Admin API
+        StUtils.waitUntilSupplierIsSatisfied(() ->
+            KafkaCmdClient.describeKafkaBrokerUsingPodCli(Environment.TEST_SUITE_NAMESPACE, scraperPodName, KafkaResources.plainBootstrapAddress(testStorage.getClusterName()), podNum).contains("unclean.leader.election.enable=" + true));
```

Comment:
Will this produce some relevant message in the wait method? Or will it be `Wait until true is satisfied`?

Reply:
Updated